site/content/en/docs/manual/advanced/analytics-and-monitoring/auto-qa.md

```diff
@@ -1,72 +1,158 @@
+That is, for the quality assurance to function correctly, the validation set must
+have some portion of the task frames, and the frames must be chosen randomly.
+
+Depending on the dataset size, data variance, and task complexity,
 **5-15% of the data is typically good enough** for quality estimation,
-while keeping extra annotation overhead acceptable.
+while keeping extra annotation overhead for the Ground Truth acceptable.
 
 For example, in a typical **task with 2000 frames**, selecting **just 5%**,
 which is 100 extra frames to annotate, **is enough** to estimate the
 annotation quality. If the task contains **only 30 frames**, it's advisable to
-select **8-10 frames**, which is **about 30%**.
+select **8-10 frames**, which is **about 30%**. It is more than 15%,
+but in the case of smaller datasets, we need more samples to estimate quality reliably,
+as data variance is higher.
+
+## Ground truth jobs
 
-It is more than 15% but in the case of smaller datasets,
-we need more samples to estimate quality reliably.
+A **Ground Truth job** (GT job) is a way to represent the validation set in a CVAT task.
+This job is similar to regular annotation jobs - you can edit the annotations manually,
+use auto-annotation features, and import annotations in this job. There can be no more
+than 1 Ground Truth job in a task.
 
-To create a **Ground truth** job, do the following:
+To enable quality estimation in a task, you need to create a Ground truth job in the task,
+annotate it, and switch the job state to be `acceptance`-`completed`.
+Once the Ground Truth job is configured, CVAT will start using this job for quality estimation.
+
+![Ground truth job actions](/images/honeypot04.jpg)
+
+### Import annotations
+
+If you want to import annotations into the Ground truth job, do the following.
+
+1. Open the task, and find the Ground truth job in the jobs list.
+2. Click on three dots to open the menu.
+3. From the menu, select **Import annotations**.
+4. Select import format, and select file.
+5. Click **OK**.
 
-1. Create a {{< ilink "/docs/manual/basics/create_an_annotation_task" "task" >}}, and open the task page.
+> **Note** that if there are imported annotations for the frames that exist in the task,
+> but are not included in the **Ground truth** job, they will be ignored.
+> This way, you don't need to worry about "cleaning up" your Ground truth
+> annotations for the whole dataset before importing them.
+> Importing annotations for the frames that are not known in the task still raises errors.
+
+### Export annotations
+
+To export annotations from the Ground Truth job, do the following.
+
+1. Open the task, and find a job in the jobs list.
+2. Click on three dots to open the menu.
+3. From the menu, select **Export annotations**.
+
+### Delete
+
+To delete the Ground Truth job, do the following.
+
+1. Open the task, and find the Ground Truth job in the jobs list.
+2. Click on three dots to open the menu.
+3. From the menu, select **Delete**.
```

Comment:
Probably need to mention that GT job created with `Honeypots` validation mode can not be deleted.

Reply:
Added